Dockerfile

```diff
@@ -0,0 +1,24 @@
+FROM node:12-buster as builder
+WORKDIR /app
+COPY package.json yarn.lock ./
+RUN export YARN_CACHE_FOLDER="$(mktemp -d)" \
+  && yarn install --frozen-lockfile --quiet \
+  && rm -r "$YARN_CACHE_FOLDER"
+COPY . .
+RUN yarn build
+
+FROM node:12-buster
```

Comment:
@mgaeta I think if you use the alpine version, it will make the overall image smaller.

Reply:
It looks like alpine worked in the test workflow!